messenger/message-recorder.rst

```diff
@@ -0,0 +1,77 @@
+.. index::
+    single: Messenger; Record messages
+
+Events Recorder: Handle Events After CommandHandler Is Done
+===========================================================
+
+Let's take the example of an application that has a command (a CQRS message) named
+``CreateUser``. That command is handled by the ``CreateUserHandler`` which creates
```

Comment:
It could be better to use `SignUpUser` as command name, it is less focus on CRUD action.